Integrations/QRadar_5_4_9/QRadar_5_4_9.py

```diff
@@ -0,0 +1,1095 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import os
+import json
+import requests
+import traceback
+import urllib
+import re
+from requests.exceptions import HTTPError
+from copy import deepcopy
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBAL VARS '''
+SERVER = demisto.params().get('server')[:-1] if str(demisto.params().get('server')).endswith('/') \
+    else demisto.params().get('server')
+CREDENTIALS = demisto.params().get('credentials')
+USERNAME = CREDENTIALS['identifier'] if CREDENTIALS else ''
+PASSWORD = CREDENTIALS['password'] if CREDENTIALS else ''
+TOKEN = demisto.params().get('token')
+USE_SSL = not demisto.params().get('insecure', False)
+AUTH_HEADERS = {'Content-Type': 'application/json'}
+if TOKEN:
+    AUTH_HEADERS['SEC'] = str(TOKEN)
+OFFENSES_PER_CALL = int(demisto.params().get('offensesPerCall', 50))
+OFFENSES_PER_CALL = 50 if OFFENSES_PER_CALL > 50 else OFFENSES_PER_CALL
+PAGE_SIZE = 200
```

Comment:
isn;t it something you've added in the new version? This file should look like the one from Master

Reply:
?

Reply:
lets make sure it's a copy paste from master, the yml as well

Reply:
done